.github/workflows/code_checks.yml

```diff
@@ -142,7 +203,7 @@ jobs:
           max_attempts: 3
           command: >
             ${{ env.COMPOSE_BASH_CMD }}
-            "bundle exec rake parallel:setup[24]"
+            "bundle exec rails db:test:prepare"
```

Comment:
Each run only has one process so we don't need 24 dbs anymore (each run has its own PG)